lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -178,15 +177,18 @@
 import re
 import time
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.ec2 import ec2_connect, ec2_argument_spec
+import ansible.module_utils.ec2 as ec2_utils
```

Comment:
We shouldn't need this import. You'll need to explicitly import boto3_conn (and possible others) but that's fairly normal